examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb

```diff
@@ -0,0 +1,517 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Taxi CrossValidation with GPU accelerating on XGBoost\n",
+    "\n",
+    "In this notebook, we will show you how to levarage GPU to accelerate taxi CrossValidation on XGBoost to find out the best model given a group parameters.\n",
+    "\n",
+    "## Import classes\n",
+    "First we need load some common classes that both GPU version and CPU version will use:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ml.dmlc.xgboost4j.scala.spark.{XGBoostRegressionModel, XGBoostRegressor}\n",
+    "import org.apache.spark.ml.evaluation.{RegressionEvaluator}\n",
+    "import org.apache.spark.ml.tuning.ParamGridBuilder\n",
+    "import org.apache.spark.sql.SparkSession\n",
+    "import org.apache.spark.sql.types.{FloatType, IntegerType, StructField, StructType}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "what is new to xgboost-spark users is rapids.GpuDataReader and **rapids.CrossValidator**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ml.dmlc.xgboost4j.scala.spark.rapids.{CrossValidator, GpuDataReader}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set dataset path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "trainParquetPath = /home/bobwang/jupyter/data/taxi/cv\n",
```

Comment:
Let's use the same path in the other files.

Reply:
changed path to : /data/mortgage/parquet/xxx